### PR TITLE
Remove sticky positioning from footer in network switcher

### PR DESCRIPTION
**Pull request by hyphenized**
Per @VladUXUI clarification, the footer in the network switcher should not be fixed at the bottom.

image

## To Test
- [x] Verify that the add network button appears at the end of the network list, rather than staying fixed at the bottom of the screen.

Latest build: [extension-builds-3246](https://github.com/tahowallet/extension/suites/12042306255/artifacts/633141626) (as of Wed, 05 Apr 2023 10:09:24 GMT).

### CODE REVIEW

File: ui/components/TopMenu/TopMenuProtocolList.tsx [hunk 1]
```diff
@@ -63,96 +63,100 @@ export default function TopMenuProtocolList({
...
-                      key={network.name}
-                      network={network}
-                      info={t("protocol.compatibleChain")}
-                      onSelect={onProtocolChange}
-                    />
-                  ))}
-                </>
-              )}
-            {showTestNetworks && testNetworks.length > 0 && (
+    <div className="">
```
Comment (kkosiorowska) [inline ui/components/TopMenu/TopMenuProtocolList.tsx:66]:
```suggestion
    <div>
```

**Review by kkosiorowska — COMMENTED** [review]
Looks nice, just one question. If the flags are disabled the footer is not at the bottom. I'm not sure if it's what we want.

Screenshot 2023-04-05 at 09 27 27

File: ui/components/TopMenu/TopMenuProtocolList.tsx [hunk 2]
```diff
@@ -63,96 +63,103 @@ export default function TopMenuProtocolList({
...
-      {customNetworksEnabled && <TopMenuProtocolListFooter />}
-    </>
+      <style jsx>
+        {`
+          .networks_list {
+            overflow-y: auto;
+            overflow-x: hidden;
+            display: flex;
+            flex-direction: column;
+            min-height: 511px;
```
Comment (kkosiorowska) [inline ui/components/TopMenu/TopMenuProtocolList.tsx:132]:
```suggestion
            min-height: 512px;
```

**Comment by hyphenized** [comment]
> Looks nice, just one question. If the flags are disabled the footer is not at the bottom. I'm not sure if it's what we want. 🤔
> 
> Screenshot 2023-04-05 at 09 27 27

Thanks, I've fixed the positioning issue. Could you re-check that the feature flag works please?

**Review by kkosiorowska — COMMENTED** [review]
It looks better! Only the `min-hight` disagrees by one pixel.

Screenshot 2023-04-05 at 11 40 58

One note after the changes, the close button disappears.

**Before**
https://user-images.githubusercontent.com/23117945/230044191-edad859b-8e4c-48af-a3af-4f8f42717491.mov

**After**
https://user-images.githubusercontent.com/23117945/230044255-b63f4600-acb5-4d4c-93aa-ccad2d82d085.mov

**Review by jagodarybacka — APPROVED** [review]
Works well, close button is visible and add button is not sticky ✅

**Pull request merged**